### ISSUE TITLE
What is the latest result in the CDR dataset

### ISSUE THREAD

**Issue by slzbywdf** [fenchri/edge-oriented-graph#13]
I use your new version code, the final result on CDR dataset F1=0.75+.  Am I wrong, what is the final result in latest version?

**Comment by fenchri** [comment]
Hi,

You need to find the best epoch based on the dev set and then train the model on train+dev and evaluate on the test. 
I got around 64% on the dev and 67% on the test set.
If you don't merge train+dev performance on the test is around 65%.

FYI, I got these results are with pytorch 1.3.1

**Comment by slzbywdf** [comment]
Thanks for your reply, my result is wrong.
